free-1.md

```diff
@@ -10,41 +10,92 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Currently, most users of LED strip lights use the default remote to control the colors
+because they don't know how to code anything better. This remote alows for a classic fade
+effect or just picking a color and leaving it there. The only way to upgrade this is by
+using somewhat tricky arduino/raspberry pi code which the average person with LED strip
+lights doesn't know. Therefore, my idea would ideally abstract the complex electrical code
+required and allow for easy customization of very fun and useful LED strip lights.  
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+To create whatever design/controls a user likes, I think a language is necessary
+because there are many features to take advantage of that a simple user interface
+could not handle. For example the user would need a way to integrate time and other
```

Comment:
I really like this idea! I think it will be important to think about the types of users you are aiming this at to be able to start to shape your language. Being able to integrate time and patterns seems really important, but if you're aiming this at users with very little technical knowledge, definitely think about ways to abstract away even looping. Like the Pavlus article we read for the previous assignment mentioned, even for loops are very difficult for beginners.